backend/actions/Model/getDashboards.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+
+module.exports = ({ db }) => async function getDashboards() {
+
+  const Model = db.models['Dashboard'];
```

Comment:
Let's move this file to `actions/Dashboard/getDashboards`. And please do `const { Dashboard } = db.models` for consistency.